pkg/build/api/types.go

```diff
@@ -562,6 +562,10 @@ type DockerBuildStrategy struct {
 	// Env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar
 
+	// Args contains any build arguments that are to be passed to Docker.  See
```

Comment:
They are consumed at different parts of the build process. Env variables are put into the build container, the Build Args would be consumed by the docker invocation. Perhaps Env isn't used by Docker Strategies? In that case, we could use Env, but I think that would be a bit confusing in the future.